hera_cal/lst_stack/metrics.py

```diff
@@ -111,9 +168,12 @@ def get_squared_zscores(
         for ipol, pol in enumerate(stack.pols):
             bl = (*ap, pol)
             data = stack.get_data(bl)
+            nsamps = stack.get_nsamples(bl)
+            M = cross_stats.nsamples[bl]
+            norm = nsamps * (M / (M - nsamps))**2
```

Comment:
might be helpful to note that `M` is the total number of nsamples over nights, while `nsamps` is a per-night quantity (if I'm understandting this correctly)

Reply:
You are indeed, and I've added a couple comments.